src/b-tabs.html

```diff
@@ -42,18 +42,16 @@
                 },
 
                 selectedChanged: function (oldValue, newValue) {
-                    var oldTab = this.getTab(oldValue), 
+                    var oldTab = this.getTab(oldValue),
                         newTab = this.getTab(newValue);
-                    
+
                     if (oldTab !== null) {
                         oldTab.classList.remove('active');
-                        oldTab.tabIndex = -1;
```

Comment:
Like I said in my "main" comment, only the currently selected tab should be in the tab order (and therefore not the b-tabs element itself)